<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartDict - 智能词典</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="placeholderText">
         <string>请输入要查询的单词...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="text">
         <string>复制结果</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contentLayout">
      <item>
       <widget class="QGroupBox" name="historyGroup">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>搜索历史</string>
        </property>
        <layout class="QVBoxLayout" name="historyVLayout">
         <item>
          <widget class="QListView" name="historyListView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="historyBtnLayout">
           <item>
            <widget class="QPushButton" name="exportButton">
             <property name="text">
              <string>导出</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="resultBrowser">
        <property name="placeholderText">
         <string>翻译结果将显示在这里...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
